components/header/NavLinks.tsx

```diff
@@ -32,23 +32,23 @@ const NavLinks: React.FC = () => {
 
 const productsDropdownLinks: IDropdownMenuLinks[] = [
   {
-    href: "https://umaproject.org/optimistic-oracle.html",
+    href: "https://umaproject.org/products.html",
     name: "Optimistic Oracle",
   },
   {
-    href: "https://umaproject.org/lsp.html",
+    href: "https://docs.umaproject.org/synthetic-tokens/long-short-pair.html",
     name: "Long Short Pair (LSP)",
   },
   {
-    href: "https://umaproject.org/call-put.html",
+    href: "https://docs.umaproject.org/products/calloption.html",
     name: "Call/Put Options",
   },
   {
-    href: "https://umaproject.org/range-tokens.html",
+    href: "https://docs.umaproject.org/range-tokens/summary.html",
     name: "Range Tokens",
   },
   {
-    href: "https://umaproject.org/kpi-options.html",
+    href: "https://docs.umaproject.org/kpi-options/summary.html",
```

Comment:
the rest of these work, but you dont need the `.html` anymore, probably more correct to remove them

Reply:
removed .html's